docs/source/post_processing.md

```diff
@@ -1,9 +1,16 @@
 # Post-processing
 
-This page shows all required steps of post-processing a `ShapePipe` run. Some details pertain specifically to
-runs carried out on [canfar](./canfar.md), but most are general.
+This page shows all required steps of post-processing the results from one or
+more `ShapePipe` runs. Post-processing combines various individual 'ShapePipe'
+output files, and creates joint results, for example combining individual tile
+catalogues in a large sky area. The output of post-processing is a joint _shape
+catalogue_, containing all required information to create a calibrated shear
+catalogue via 'metacalibration'), a joint star catalogue, and PSF diagnostic plots.
```

Comment:
I would propose `'metacalibration'` -> `_metacalibration_`. Single quotes don't render very nicely in this theme.

Reply:
Changed.